scout/server/blueprints/variants/controllers.py

```diff
@@ -833,120 +833,154 @@ def callers(variant_obj, category='snv'):
+    breakpoint_2 = None
+    if variant_obj.get('end_chrom'):
+        breakpoint_2 = str(variant_obj.get('end_chrom'))+':'+str(variant_obj.get('end'))
+    else:
+        breakpoint_2 = str(variant_obj.get('chromosome'))+':'+str(variant_obj.get('end'))
+
+    variant_size = variant_obj.get('length')
+    panels = ', '.join(variant_obj['panels'])
     hgnc_symbol = ', '.join(variant_obj['hgnc_symbols'])
     gtcalls = ["<li>{}: {}</li>".format(sample_obj['display_name'],
                                         sample_obj['genotype_call'])
                for sample_obj in variant_obj['samples']]
-    tx_changes = []
-    for gene_obj in variant_obj.get('genes', []):
-        for transcript_obj in gene_obj['transcripts']:
-            parse_transcript(gene_obj, transcript_obj)
-            if transcript_obj.get('change_str'):
-                tx_changes.append("<li>{}</li>".format(transcript_obj['change_str']))
-
-    html = """
-      <ul">
-        <li>
-          <strong>Case {case_name}</strong>: <a href="{url}">{variant_id}</a>
-        </li>
-        <li><strong>HGNC symbols</strong>: {hgnc_symbol}</li>
-        <li><strong>Gene panels</strong>: {panels}</li>
-        <li><strong>GT call</strong></li>
-        {gtcalls}
-        <li><strong>Amino acid changes</strong></li>
-        {tx_changes}
-        <li><strong>Ordered by</strong>: {name}</li>
-      </ul>
-    """.format(case_name=case_obj['display_name'],
-               url=variant_link,
-               variant_id=variant_obj['display_name'],
-               hgnc_symbol=hgnc_symbol,
-               panels=', '.format(variant_obj['panels']),
-               gtcalls=''.join(gtcalls),
-               tx_changes=''.join(tx_changes),
-               name=user_obj['name'].encode('utf-8'))
-
-    kwargs = dict(subject="SCOUT: Sanger sequencing of {}".format(hgnc_symbol),
-                  html=html, sender=sender, recipients=recipients,
-                  # cc the sender of the email for confirmation
-                  cc=[user_obj['email']])
-
-    # compose and send the email message
-    message = Message(**kwargs)
-    mail.send(message)
-
-    store.order_sanger(institute_obj, case_obj, user_obj, variant_link, variant_obj)
-
+    tx_changes = 'Not available'
+
+    if category == 'snv': #SNV
+        view_type = 'variants.variant'
+        display_name = variant_obj.get('display_name')
+        tx_changes = []
+        for gene_obj in variant_obj.get('genes', []):
+            for transcript_obj in gene_obj['transcripts']:
+                parse_transcript(gene_obj, transcript_obj)
+                if transcript_obj.get('change_str'):
+                    tx_changes.append("<li>{}</li>".format(transcript_obj['change_str']))
+        tx_changes=''.join(tx_changes)
+
+    else: #SV
+        view_type = 'variants.sv_variant'
+        display_name = breakpoint_1+'_'+variant_obj.get('sub_category').upper()
+
+    # body of the email
+    html = verification_email_body(
```

Comment:
Please use argument names when this many. It is easy that one ends up in the wrong parameter when using these many arguments